From da8a6f171553d3f05138a6abfe32929985f56116 Mon Sep 17 00:00:00 2001
From: Giuseppe Barillari <joe2k01dev@gmail.com>
Date: Thu, 19 Apr 2018 20:12:34 +0200
Subject: [PATCH 3/5] wiki: recovery_install_heimdall: Don't make the users
 flash TWRP over boot partition

 * Specify that they need to hold recovery boot keys as soon as Odin finishes
   flashing TWRP.

Change-Id: I0cd94618107ff73e76f8f963f836df1a0b94b0ce
Signed-off-by: Giuseppe Barillari <joe2k01dev@gmail.com>
---
 _includes/templates/recovery_install_heimdall.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/_includes/templates/recovery_install_heimdall.md b/_includes/templates/recovery_install_heimdall.md
index 3925d75..ddc3865 100644
--- a/_includes/templates/recovery_install_heimdall.md
+++ b/_includes/templates/recovery_install_heimdall.md
@@ -57,8 +57,8 @@ The preferred method of installing a custom recovery is through this boot mode{%
     Accept the disclaimer, then insert the USB cable into the device.
 4. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window in the directory the recovery image is located, and type:
 
-        heimdall flash --BOOT twrp-x.x.x-x-{{ twrp_codename }}.img --RECOVERY twrp-x.x.x-x-{{ twrp_codename }}.img
+        heimdall flash --RECOVERY twrp-x.x.x-x-{{ twrp_codename }}.img
 
     {% include tip.html content="The file may not be named identically to what stands in this command, so adjust accordingly. If the file is wrapped in a zip or tar file, extract the file first, because Heimdall is not going to do it for you." %}
 5. A blue transfer bar will appear on the device showing the recovery being transferred.
-6. The device will reboot into recovery.
+6. As soon as Odin finishes flashing TWRP hold your device recovery boot keys to boot in recovery mode.
-- 
2.17.0

